Agregando la pagina Donate

Morban15 opened the pull request:
En esta pagina ponemos en prectica las templanzas, por lo que la hemos
agregado para los donantes.

### Issues

_¿Qué se está resolviendo?_
Debemos agregar una página para donantes en español
y en inglés.

### Todos

_¿Qué queda por hacer?_
Revisar y terminar

### Áreas impactadas

_¿Qué páginas o componentes están afectados?_
index.html

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

Morban15 committed:
Agregando la pagina Donate

En esta pagina ponemos en prectica las templanzas, por lo que la hemos
agregado para los donantes.

Morban15 committed:
Agregando la pagina Donate

En esta pagina ponemos en prectica las templanzas, por lo que la hemos
agregado para los donantes.

misscs commented on app/views/donate/index.html:
```diff
@@ -0,0 +1,13 @@
+{% extends "layout/_simple_site_layout.html" %}
+
+{% set data = donate %}
```
Esto se puede remover ya que sabemos que podemos usar la llave del objeto :D

misscs commented on app/views/home/index.html:
```diff
@@ -4,7 +4,8 @@
+<p>{{ home.headline }}</p>
 
 </article>
 {% endblock %}
```
Cómo estos cambios están el la pagina `home` debía haber sido outro commit porque no está relacionada con agregar la página donate. 

No se olviden remover la linea 3. Entiende porque lo deben remover? (Este cambio se puede combinar con el otro cambio)

misscs reviewed (changes requested):
Muy bien trabajo! Dos comentarios chiquitos.

Morban15 committed:
Quitando cosas no necesarias en index.html

Hemos removido h1, porque es opcional y no lo queríamos tener en nuestra
página.

Morban15 committed:
Merge branch 'donate' of https://github.com/cosola-macile/macile.org into donate

Morban15 committed:
Corrigiendo errores en index.html

Hemos removido los errores de index.html de la pagina donate.
